gradle.properties

```
@@ -26,7 +26,7 @@ android.enableJetifier=true
 
 GROUP=com.amazonaws
 
-VERSION_NAME=2.22.7
+VERSION_NAME=2.22.8
```

Custom endpoints is a new feature, so I think we should bump the minor version.  The reason that the script didn't bump the minor version is because the commit message has a typo - it starts with `"feat` instead of `feat`

```suggestion
VERSION_NAME=2.23.0
```